test/langtools/tools/javac/patterns/SwitchErrors.java

```diff
@@ -185,4 +185,9 @@ Object guardWithMatchingExpression(Object o1, Object o2) {
             default -> null;
         };
     }
+    void exhaustiveAndNull(String s) {
+        switch (s) {
+            case null: break;
+        }
+    }
```

Comment:
The other possible new syntax is "case default", but there is no error because it's exhaustive.

```
void exhaustiveCaseDefaultNoError(String s) {
        switch (s) {
            case default: break;
        }
    }
```

Reply:
Correct, there are tests that verify (at least as a side-effect) that `case default` is total in `test/langtools/tools/javac/patterns/CaseDefault.java`.